terraform/paas/application.tf

```diff
@@ -15,8 +15,8 @@ resource "cloudfoundry_app" "app_application" {
   }
 
   docker_credentials = {
-    username = var.docker_username
-    password = var.docker_password
+    username = jsondecode(data.azurerm_key_vault_secret.DevOps.value)["DOCKER_USERNAME"]
```

Comment:
could assign `jsondecode(data.azurerm_key_vault_secret..)` to a local variable and reuse?

Reply:
Indeed. Something like:
```
devops_secrets = jsondecode(data.azurerm_key_vault_secret.DevOps.value)
app_secrets = jsondecode(data.azurerm_key_vault_secret.Application.value)
username = devops_secrets["DOCKER_USERNAME"]
...
environment = {
    HTTPAUTH_PASSWORD = app_secrets["HTTPAUTH_PASSWORD"]
...
```

Reply:
I get what you are saying